localflavor/nl/models.py

```diff
@@ -82,3 +82,28 @@ def formfield(self, **kwargs):
         defaults = {'form_class': forms.NLBSNFormField}
         defaults.update(kwargs)
         return super(NLBSNField, self).formfield(**defaults)
+
+
+class NLLicensePlateField(models.CharField):
+    """
+    A Dutch car license plate.
```

Comment:
It's probably better to remove 'car' as this can be used for all Dutch license plates.